test/NuGet.Core.Tests/NuGet.CommandLine.XPlat.Test/BasicLoggingTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.CommandLine.XPlat.Test
+{
+    public class BasicLoggingTests
+    {
+        [Fact]
+        public void BasicLogging_VerifyExceptionLogged()
+        {
+            // Arrange
+            var log = new TestLogger();
+            Program.Log = log;
+
+            var args = new string[]
+            {
+                "--unknown",
+            };
+
+            // Act
+            var exitCode = Program.Main(args);
+
+            // Assert
+            Assert.Equal(1, exitCode);
+            Assert.Equal(2, log.Messages.Count);
+            Assert.Equal(1, log.Errors);
+            Assert.Equal(0, log.Warnings);
+            Assert.Contains("--unknown", log.Messages.ToArray()[0]);  // error
+            Assert.Contains("Program.cs", log.Messages.ToArray()[1]); // verbose stack trace
+        }
+
+        [Fact]
+        public void BasicLogging_NoParams_ExitCode()
+        {
+            // Arrange
+            var log = new TestLogger();
+            Program.Log = log;
+
+            var args = new string[]
+            {
+                //empty
+            };
+
+            // Act
+            var exitCode = Program.Main(args);
+
+            // Assert
+            Assert.Equal(0, exitCode);
+        }
+
+        [Fact]
+        public void BasicLogging_RestoreHelp_ExitCode()
+        {
+            // Arrange
+            var log = new TestLogger();
+            Program.Log = log;
+
+            var args = new string[]
+            {
+                "restore",
+                "--help",
+            };
+
+            // Act
+            var exitCode = Program.Main(args);
+
+            // Assert
+            Assert.Equal(0, exitCode);
+        }
+
+        [Fact]
+        public void BasicLogging_RestoreUnknownOption_ExitCode()
```

Comment:
Is it possible to test a case where the command returns status code < 0 || > 255? I do not know the feasibility but it would be good to verify that we don't regress the override to exit code 1.